waku/v2/peermanager/peer_connector.go

```diff
@@ -173,6 +180,10 @@ func (c *PeerConnectionStrategy) isPaused() bool {
 	return c.paused.Load()
 }
 
+func (c *PeerConnectionStrategy) SetPause(paused bool) {
```

Comment:
Should the name be `SetPaused` to be consistent with the name of the attribute being set?

Reply:
yes